test/backtrace.jl

```diff
@@ -221,7 +228,6 @@ let trace = try
     end
     @test trace[1].func === Symbol("top-level scope")
     @test trace[1].file === :a_filename
-    @test trace[1].line == 2
```

Comment:
I think this can be brought back; seems to work.

Reply:
Will fix